src/overlays/actors/ovl_Bg_Spot18_Basket/z_bg_spot18_basket.c

```diff
@@ -21,39 +39,396 @@ const ActorInit Bg_Spot18_Basket_InitVars = {
     (ActorFunc)BgSpot18Basket_Update,
     (ActorFunc)BgSpot18Basket_Draw,
 };
-*/
-#pragma GLOBAL_ASM("asm/non_matchings/overlays/actors/ovl_Bg_Spot18_Basket/func_808B7710.s")
 
-#pragma GLOBAL_ASM("asm/non_matchings/overlays/actors/ovl_Bg_Spot18_Basket/func_808B7770.s")
+static ColliderJntSphItemInit sJntSphItemsInit[2] = {
+    {
+        { 0x00, { 0x00000000, 0x00, 0x00 }, { 0x00000000, 0x00, 0x00 }, 0x00, 0x00, 0x01 },
+        { 0, { { 0, 2040, 0 }, 54 }, 100 },
+    },
+    {
+        { 0x00, { 0x00000000, 0x00, 0x00 }, { 0x00000008, 0x00, 0x00 }, 0x00, 0x01, 0x00 },
+        { 1, { { 0, 1400, 0 }, 13 }, 100 },
+    },
+};
+
+static ColliderJntSphInit sJntSphInit = {
+    { COLTYPE_UNK10, 0x00, 0x09, 0x09, 0x20, COLSHAPE_JNTSPH },
+    2,
+    sJntSphItemsInit,
+};
+
+static s16 D_808B85C8[] = { 32768, 10922, 54613, 0 };
+
+void func_808B7710(Actor* thisx, GlobalContext* globalCtx) {
+    BgSpot18Basket* this = THIS;
+
+    Collider_InitJntSph(globalCtx, &this->colliderJntSph);
+    Collider_SetJntSph(globalCtx, &this->colliderJntSph, &this->dyna.actor, &sJntSphInit, &this->colliderJntSphItem);
+    this->dyna.actor.colChkInfo.mass = 255;
+}
+
+void func_808B7770(BgSpot18Basket* this, GlobalContext* globalCtx, f32 arg2) {
+    Vec3f acceleration;
+    Vec3f velocity;
+    Vec3f position;
+    f32 cosValue;
+    s32 i = 0;
+    f32 randomValue;
+    static s16 D_808B85D0 = 0;
+    f32 sinValue;
+    s32 arraySize = 2;
+
+    while (i != arraySize) {
+        if (globalCtx) {}
+        if (!(arg2 < Math_Rand_ZeroOne())) {
+            D_808B85D0 += 30000;
```

Comment:
`D_808B85D0` is a rotation (argument for sins/cos), so use hex for `30000` (as heartbreaking as it is)